### ISSUE TITLE
Doesn't work with manage.py in /src folder

### ISSUE THREAD

**Issue by emacstheviking** [Pachwenko/VSCode-Django-Test-Runner#20]
I configured my django.

"python.djangoTestRunner.prefixCommand": "cd /Users/sean/PyCharmProjects/API/src &&"

This makes it change to the correct folder to run manage.py but the file path generated always starts with 'src' which is incorrect given I am already in src, it is never going to work.

**Comment by KerberosMorphy** [comment]
> Ha, so I find myself back here again 15 months later as there are still no other decent ways to integrate Django testing. You get what you pay for....... time to start hacking.....

Did you start hacking? Because I'm about to. It almost works except that it execute `manage.py test src.something.something_else` while I need `manage.py test something.something_else`

**Comment by rozbr96** [comment]
I came across this issue just now. I had to alter the extension code a bit more, but now it's working nicely

**Comment by emacstheviking** [comment]
Nice! So do I just reinstall the extension??

**Comment by emacstheviking** [comment]
> > Ha, so I find myself back here again 15 months later as there are still no other decent ways to integrate Django testing. You get what you pay for....... time to start hacking.....
> 
> Did you start hacking? Because I'm about to. It almost works except that it execute `manage.py test src.something.something_else` while I need `manage.py test something.something_else`

No, sadly. Day job way too demanding atm.b :(

**Comment by rozbr96** [comment]
> Nice! So do I just reinstall the extension??

Nope. At least, not from the VSCode extensions store. You can, however, try one of the code wrote by other guys.

[There is this pull request](https://github.com/Pachwenko/VSCode-Django-Test-Runner/pull/4) and [this another one](https://github.com/Pachwenko/VSCode-Django-Test-Runner/pull/8).

I believe any of them would be enough.

Just clone the desired repo, compile, and install it using the VSCode `Install Extension from Location...` command. That's what I did.
